src/App.js

```diff
@@ -0,0 +1,19 @@
+import { Route, Routes } from 'react-router-dom';
+import Header from './components/Header';
+import './App.css';
```

Comment:
- [optional] please remember this is a mobile version project. Ensure it's responsive till 768px where it switches. From 580px it switches already.
- [optional] kindly add the small arrows on the cards, so it indicates that it should be clicked and as well follow the original design
- [optional] please increase the size of the search bar for user-friendly purposes.
![Screenshot (1287)](https://github.com/Gh-Reza/weather-app/assets/65298048/41438272-06b2-47fa-80e5-13e4af526c82)
![Screenshot (1288)](https://github.com/Gh-Reza/weather-app/assets/65298048/3eede611-f6c7-4a9a-a2a0-f1b99215c76f)